spec/lib/identity_job_log_subscriber_spec.rb

```diff
@@ -412,7 +406,7 @@
           trace_id: nil,
           queue_name: kind_of(String),
           job_id: job.job_id,
-          exception_class_warn: {},
+          exception_class_warn: 'Errno::ECONNREFUSED',
```

Comment:
this corresponds to https://github.com/18F/identity-idp/pull/5973#discussion_r810331662